packages/calling-component-bindings/tsconfig.preprocess.json

```diff
@@ -2,7 +2,6 @@
   "extends": "../../common/config/tsc/tsconfig.json",
   "compilerOptions": {
     "baseUrl": "./preprocessed",
-    "noImplicitAny": false,
```

Comment:
so this deletion implicitly makes it true? thats ironic

Reply:
Yes! This line can be removed everywhere; strict mode enables this and the override is no longer needed.